src/helpers/label.ts

```diff
@@ -53,3 +56,65 @@ export const getLabel = async (name: string): Promise<boolean> => {
 
   return false;
 };
+
+// Function to update labels based on the base rate difference
+export async function updateLabelsFromBaseRate(owner: string, repo: string, context: Context, labels: Label[], previousBaseRate: number) {
```

Comment:
Here's a nitpick in terms of coding style. 
```suggestion
export const updateLabelsFromBaseRate = async (owner: string, repo: string, context: Context, labels: Label[], previousBaseRate: number) =>{
```